docs/index.md

```diff
@@ -3,13 +3,10 @@
 page_title: "harvester Provider"
 subcategory: ""
 description: |-
-  
```

Comment:
All files in docs were generate by `go generate`, should not change them manually